index.bs

```diff
@@ -1889,6 +1935,141 @@ The [=remote end steps=] with |session| and <var ignore>command parameters</var>
 
 </div>
 
+#### The browser.createUserContext Command #### {#command-browser-createUserContext}
+
+The <dfn export for=commands>browser.createUserContext</dfn> command creates a [=user context=].
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      browser.CreateUserContext = (
+        method: "browser.createUserContext",
+        params: EmptyParams,
+      )
+      </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl local-cddl">
+      browser.createUserContextResult = {
+        userContext: browser.UserContext
+      }
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm="remote end steps for browser.createUserContext">
+
+The [=remote end steps=] are:
+
+1. Let |user context| be the result of performing implementation defined steps to create an [=empty user context=].
```

Comment:
I think you can just say `Let |user context| be the result of creating an [=empty user context=]`, or something like that. At some level this clearly must involve implementation-specific processing, but I don't think we need to call it out in this case.

Reply:
I think being explicit here is important. For example, I just started looking at this PR and if I didn't see "implementation-defined", I'd be questioning how to create an [=empty user context=] to matrix :P